### PR TITLE
fixed poster css property

### PR DESCRIPTION
**Pull request by zabellfour**

### CODE REVIEW

**Comment by m90** [comment]
Thanks for taking care of this. Two questions:

- can you confirm that this fixes the issue for you?
- could you undo the changes in the minified files? We should release this as 3.1.2 - I  can take care of this once merged.

**Comment by zabellfour** [comment]
- Yes I can confirm evrething works well in all browsers.
- Changes in minify JS files undone.

File: dist/seeThru.min.js [hunk 1]
```diff
@@ -1,2 +1,2 @@
-/*! seethru 3.1.1 14-08-2017 see https://github.com/m90/seeThru for details */
```
Comment (m90) [inline dist/seeThru.min.js]: Can we undo these changes as well so that the diff covers `src/seeThru.js` only? Thanks.

Comment (zabellfour) [inline dist/seeThru.min.js]: Sory, now it fixed.

**Comment by m90** [comment]
Amazing, thanks!

**Comment by m90** [comment]
Published to npm as 3.1.2
